runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -11,6 +11,7 @@ local language = require('vim.treesitter.language')
 ---@field _regions table List of regions this tree should manage and parse
 ---@field _lang string Language name
 ---@field _regions table
+---@field _custom_injections table<string, table> table of lang to list of regions
```

Comment:
```suggestion
---@field _custom_injections table<string, table> table of lang to list of regions
```
(while you're touching that code)

Wouldn't using `regions` be better here?